esp/esp/program/modules/base.py

```diff
@@ -512,28 +512,56 @@ def _checkGrade(moduleObj, request, tl, *args, **kwargs):
 
 # Just broke out this function to allow combined deadlines (see meets_any_deadline,
 # meets_all_deadlines functions below).  -Michael P, 6/23/2009
+
+# Combined deadlines are annoying; meets_deadline has been absorbed into
+# meets_any_deadline and meets_all_deadlines has been deleted --bpchen,
+# 2017-02-17
```

Comment:
You can just get rid of these two comments, IMO.  The history is all in git if we need it, and I don't think it helps with understanding current code.